agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -181,7 +183,7 @@ type DockerClient interface {
 
 	// ListPlugins returns the set of docker plugins installed on the host. A timeout value should be provided for
 	// the request.
-	ListPlugins(context.Context, time.Duration) ListPluginsResponse
```

Comment:
Could you please add a `TODO` that `ListPluginsWithFilters` can be removed in the future since Docker API calls take in filters as an arg?